test/resources/grcs_8.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
copyright notice

Reply:
Should I include that? The other file in this folder also doesn't have it.

Reply:
Whoops, might've commented on wrong file